### ISSUE TITLE
/1.4.0/Chapter06/2.2 fine_tuning.ipynb 全连接层参数名是否有误？

### ISSUE THREAD

**Issue by AskyJx** [caicloud/tensorflow-tutorial#105]
TRAINABLE_SCOPES='InceptionV3/Logits,InceptionV3/**AuxLogit**'
是否应为
TRAINABLE_SCOPES='InceptionV3/Logits,InceptionV3/**AuxLogits**'
